### PR TITLE
chore: clean extra translations

### PR DESCRIPTION
**Pull request by fmiras**
Tests on master are not working cause there are extra keys on non-default locales.

When keys on `en.json` are deleted and translate script is executed it only adds new keys to other locales but doesn't delete extra ones.

This PR adds the clean functionality.

### CODE REVIEW

File: scripts/translate.js [hunk 1]
```diff
@@ -80,9 +80,19 @@ async function main() {
       __dirname,
       `../src/Translation/locales/${locale}.json`
     )
-    const currentTranslations = require(localePath)
+    const currentTranslations = flat(require(localePath))
+    // remove obsolete keys
+    const translations = await translation.fetch(DEFAULT_LOCALE)
```
Comment (NicoSantangelo) [inline scripts/translate.js]: I think this is already available as `mainTranslations`? If not, I think it should be fetched outside the for loop, because it shouldn't change

Comment (NicoSantangelo) [reply scripts/translate.js]: Moreover, can't we perform this process on the main loop above? where `const translations = await translation.fetch(locale)`

The script is a bit hard to follow to me (not you'r code, what's already there), so if you can't or it's too hard don't worry about it

Comment (fmiras) [reply scripts/translate.js]: I also got a little bit confused when reading this code. This is what I could do without to much refactoring: 50dfb4a
